lib/game.rb

```diff
@@ -6,30 +7,36 @@ class Game
   def initialize(input:, output:)
     @input = input
     @output = output
+    @grid = Grid.new
```

Comment:
Nice. I like that I don't need to think about this (at this level). I can just understand and trust that this does its  work.